examples/custom_outs/BUILD.bazel

```diff
@@ -9,7 +9,9 @@ load("@aspect_rules_swc//swc:defs.bzl", "swc_transpiler")
             "--config",
             "module.type=" + format,
         ],
-        js_outs = [format + "/out." + ("cjs" if format == "commonjs" else "js")],
+        # The extension of the outputs can be modified using js_outs
+        js_outs = [format + "/in." + ("cjs" if format == "commonjs" else "js")],
```

Comment:
Why rename the output file to "in"?

Reply:
The input file is `in.ts` so the only option is `in.js` or with to change the extension to `in.cjs`. We can't have blanket output file renames anymore since we can't rely on index matching up

Reply:
I'll change it to `a.ts` so the example is less confusing